package.json

```diff
@@ -64,15 +64,13 @@
     "po2json": "0.4.5",
     "postcss": "7.0.14",
     "probe-image-size": "4.0.0",
-    "regenerator-runtime": "0.13.2",
```

Comment:
I think is was a dependency of `@babel/plugin-transform-runtime` and used by many other packages. I suppose we pinned it because of some dependency problems in the past so it should be safe.

Let's double check by building an add-ons linter build and try to run it.

Reply:
Tested and verified that the linter still works.

Reply:
Actually, we do need this for the integration test to pass...